### PR TITLE
NE-753: Update the branch name

### PR DESCRIPTION
**Pull request by xenolinux**
A minor update as a part of [NE-753](https://issues.redhat.com/browse/NE-753)
OCP version 4.10
QE contact @lihongan PTAL
original PR https://github.com/openshift/openshift-docs/pull/41811

### CODE REVIEW

**Comment by alebedev87** [comment]
/lgtm

Thanks for the PR, @xenolinux !

**Comment by lihongan** [comment]
LGTM
Thank you @xenolinux

File: networking/external_dns_operator/nw-installing-external-dns-operator.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ You can install the External DNS Operator using the Red Hat OpenShift Container
 
 * You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
 * You have created the operand `external-dns` namespace by running `oc create ns external-dns` command.
-* You have granted the operator access to manage operand resources by running `oc apply -f https://raw.githubusercontent.com/openshift/external-dns-operator/main/config/rbac/extra-roles.yaml` command.
+* You have granted the operator access to manage operand resources by running `oc apply -f https://raw.githubusercontent.com/openshift/external-dns-operator/release-0.1/config/rbac/extra-roles.yaml` command.
```
Comment (bergerhoffer) [inline networking/external_dns_operator/nw-installing-external-dns-operator.adoc:15]: It's fine for now, but this URL really shouldn't be rendering as a link. Can you consider an update that stops this from rendering as a clickable link?

I see a few other minor pieces of guideline/formatting things in this file that should be updated, so I can follow up with you offline after GA with some suggestions if that's okay!

**Review by bergerhoffer — COMMENTED** [review]
Added a note for later, but okay with LGTMing this PR with this update!

File: networking/external_dns_operator/nw-installing-external-dns-operator.adoc (hunk 1 above)
Comment (xenolinux) [reply networking/external_dns_operator/nw-installing-external-dns-operator.adoc:15]: Yes, sure I will make some updates to enhance the content. Thank you

**Pull request merged**